### ISSUE TITLE
Question about Fine-tuning

### ISSUE THREAD

**Issue by anonymous-pusher** [IVRL/VolRecon#11]
Hello, thank you for sharing the code of your great work.
I was wondering if it's possible to fine-tune the pretrained model on a new scene rather than running inference, similar to how it is done in SparseNeuS. In the paper, there was no mention of this so I just assumed that it is not possible but I might be wrong.

Also, I was trying to run the model on a scene from BlendedMVS, but I could not get any meaningful reconstruction as seen here:
![image](https://github.com/IVRL/VolRecon/assets/91318638/7084e801-2089-4107-b3e9-2cecbfef9747)

when looking at the rendered depth maps, I get something, but the mesh generation does not give good result:
![image](https://github.com/IVRL/VolRecon/assets/91318638/42111878-c13c-47bf-a8f7-841545699a59)
 I tried tuning the value of self.offset_dist but still no success. Do you have an idea of what could be wrong here ?

Thanks

**Comment by wtishere** [comment]
I encountered the same problem, any solutions?

**Comment by FangjinhuaWang** [comment]
Hi, sorry for the late reply. For finetuning, we tried finetuning with rendering loss before, but it did not improve the performance. So finetuning needs more geometric supervisions, such as patch warping in neuralwarp. However, when we tried finetuning SparseNeuS, we find that this could be unstable sometimes and finetuning may fail (this is also reported from others: https://github.com/xxlong0/SparseNeuS/issues/17). Therefore, we did not finetune.
